<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Календарь дней рождения</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QFrame" name="frame_Widgets">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="tb_Vacation">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/src/resources/Icon/image.png</normaloff>:/src/resources/Icon/image.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tb_Chat">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/src/resources/Icon/chat.png</normaloff>:/src/resources/Icon/chat.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="cb_Vacation">
            <item>
             <property name="text">
              <string>Январь</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Февраль</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Март</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Апрель</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Май</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Июнь</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Июль</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Август</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Сентярбь</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Октябрь</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ноябрь</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Декабрь</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tb_Style">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/src/resources/Icon/sun.png</normaloff>
              <selectedon>:/src/resources/Icon/sunBlack.png</selectedon>:/src/resources/Icon/sun.png</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QTableView" name="tableView_Vacation"/>
        </item>
        <item row="3" column="0">
         <widget class="ChatWidget" name="chatWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_DataBase">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pb_Add">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string> Добавить</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/src/resources/Icon/plus.png</normaloff>:/src/resources/Icon/plus.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Del">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string> Удалить</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/src/resources/Icon/cancel.png</normaloff>:/src/resources/Icon/cancel.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Save">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string> Сохранить</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/src/resources/Icon/save.png</normaloff>:/src/resources/Icon/save.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Sort">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string> Сортировать</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/src/resources/Icon/text-alignment.png</normaloff>:/src/resources/Icon/text-alignment.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Send">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string> Отправить</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/src/resources/Icon/follow-up.png</normaloff>:/src/resources/Icon/follow-up.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pb_Send_Shadow">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Фон.Отравка</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QWidget</extends>
   <header location="global">Chat/ChatWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
